drivers/network/afd/afd/read.c

```diff
@@ -177,14 +177,16 @@ static NTSTATUS ReceiveActivity( PAFD_FCB FCB, PIRP Irp ) {
                            RecvReq->BufferCount, FALSE );
             if (FCB->Overread && FCB->LastReceiveStatus == STATUS_SUCCESS)
```

Comment:
This is the only use of the `Overread` member. Please remove the member entirely.